ui/raidboss/data/06-ew/raid/p3n.txt

```diff
@@ -0,0 +1,110 @@
+## Pandaemonium: Asphedolos, the Third Circle
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 100000 jump 0
+
+# -ii 669F 66A7 66A8 66A0 66A1 66AF 66B0 66B3 6D0C 6DDC 6EDD
+# -ic "Liturgic Bell"
+# -it Phoinix
+# -p 66AD:118 66A3: 200
+
+0.0 "--sync--" sync /Engage!/ window 0,1
```

Comment:
We can sync on the first hit:
`3.8 "--sync--" sync / 1[56]:[^:]*:Phoinix:368:/ window 4.3,0`

Reply:
I find initial auto syncs to often be very inconsistent.  I think better to just have a starts casting on the first ability, for consistency.